Straszkiewicz_TechWr.md

```diff
@@ -0,0 +1,139 @@
+# Sentiment Analysis Tool
+#### Kinga Straszkiewicz
+##### June 2019
+
+
+
+## Introduction
+Sentiment analysis is the automated process of understanding an opinion about a given subject from written or spoken language.
+
+Also known as opinion mining or emotion AI, it refers to the use of natural language processing, text analysis, computational linguistics, and biometrics to systematically identify, extract, and study affective states and subjective information.
+
+Sentiment analysis is widely applied to voice of the customer materials such as reviews and survey responses, online and social media, and healthcare materials for applications that range from marketing to customer service to clinical medicine.
+
+## Description
+Sentiment Analysis Tool is a software designed for sentiment analysis, involving statistical examination of a given text.
+
+### Features:
+* The analysis of overall sentiment of the text
+* Providing the number of positive and negative words in the text, including stopwords
+* Providing the number of positive and negative words in the text without stopwords
+* Ratio of the positive/negative sentences to the whole text
+* The frequency of appearance of positive and negative words
+* The statistical analysis of occurrence of the sentiment words
+* Sentiment analysis on the basis of sentences containing a keyword
+* Viewing the data both as text and graphs
+
+### Setup
+The script of this tool is written in Python 3, so it is vital to have on the computer a program capable of processing this programming language.
+**The programme also requires the installation of three Python modules**:
+* re
+* matplotlib.pyplot
+* collections
+
+For the Sentiment Analysis Tool to work properly, you need to download a few additional files:
+* ._txt_ file with the first text meant for analysis
+* ._txt_ file with the second text meant for analysis
+* ._txt_ file with the list of positive words
+* ._txt_ file with the list of negative words
+* ._txt_ file with the list of stopwords
+
+They all have to be located in the same folder as the programme.
+
+To run, download a zip file:
+
+https://github.com/kingastraszkiewicz/Project-Sentiment-Analysis/blob/master/Projekt.zip
+
+
+## User Guide
+After launching the tool, there appears the Menu displaying the contents of the programme and the numbers, which are ascribed to the options.
+The first thing that you have to do is to pick one of the two texts meant for analysis.
+
+![alt text](https://github.com/kingastraszkiewicz/Project-Sentiment-Analysis/blob/master/2019-06-10_17h49_01.png)
```

Comment:
Your images do not render properly ;( You are linking the so-called "blob", please try a normal link instead.